shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -19,6 +19,26 @@ enum ResultValue {
     Varchar(String),
     Int(i32),
     Boolean(bool),
+    Uuid(uuid::Uuid),
+    Ascii(String),
+    BigInt(i64),
+    Blob(Vec<u8>),
+    Decimal(Vec<u8>),
+    Double(Vec<u8>),
+    Duration(Vec<u8>),
+    Float(Vec<u8>),
```

Comment:
Should we store this as a float?
And there seems to be other similar cases too.

Reply:
Yeah, I'll switch them over.